.github/support.md

```diff
@@ -0,0 +1,21 @@
+## Are you having trouble getting started?
+If you haven't already, please checkout our [documentation](https://docs.bugsnag.com/platforms/ios/symbolication-guide/#using-cocoapods-plugin) for guides, references and tutorials.
+
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour 
+
+* Ruby version:
+* Bundler version:
+* CocoaPods version:
+
+## Bug or Feature Requests
+If you would like to raise a bug or feature request please do so by creating a [New Issue](https://github.com/bugsnag/cocoapods-bugsnag/issues/new/choose) and selecting bug or feature.
+Please note: we cannot promise that we will fulfil all requests
+
+## Pull Requests
+If you have made a fix and would like to raise a pull request, please read our [CONTRIBUTING.md](../CONTRIBUTING.md) file before creating the pull request.
```

Comment:
This repo doesn't have a `CONTRIBUTING.md` doc, so this link is broken. Discovered this by accident earlier!